tests/scipy_stats_test.py

```diff
@@ -913,5 +913,41 @@ def evaluate_kde(kde, x):
     tree_util.tree_map(lambda a, b: self.assertAllClose(a, b), kde, kde2)
     self.assertAllClose(evaluate_kde(kde, x), kde.evaluate(x))
 
+  @parameterized.named_parameters(jtu.cases_from_list(
```

Comment:
We're gradually moving the tests to use `jtu.sample_product` instead of `parameteriozed.named_parameters` (see #12660). You could optionally do that here if you're feeling up for it! But if not, this looks fine.